__tests__/lint/assertions-error/snapshot.js

```diff
@@ -0,0 +1,328 @@
+
+No configurations were defined in extends -- using built in recommended configuration by default.
+
+validating /openapi.yaml...
+[1] openapi.yaml:17:5 at #/tags/0/description
+
+Tag description must be at least 13 characters and end with a full stop.
+
+15 | tags:
+16 |   - name: test
+17 |     description: Description.
+   |     ^^^^^^^^^^^
+18 |   - name: about
+19 |     description: Everything about your Pets
+
+Error was generated by the assertions rule.
+
+
+[2] openapi.yaml:19:5 at #/tags/1/description
+
+Tag description must be at least 13 characters and end with a full stop.
+
+17 |     description: Description.
+18 |   - name: about
+19 |     description: Everything about your Pets
+   |     ^^^^^^^^^^^
+20 |   - name: Without description
+21 | paths:
+
+Error was generated by the assertions rule.
+
+
+[3] openapi.yaml:20:5 at #/tags/2/description
+
+Every tag must have a name and description.
+
+18 |   - name: about
+19 |     description: Everything about your Pets
+20 |   - name: Without description
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^
+21 | paths:
+22 |   /pet/findByStatus:
+
+Error was generated by the assertions rule.
+
+
+[4] openapi.yaml:22:3 at #/paths/~1pet~1findByStatus/get
+
+Every path item must have a GET operation.
+
+20 |   - name: Without description
+21 | paths:
+22 |   /pet/findByStatus:
+   |   ^^^^^^^^^^^^^^^^^
+23 |     put:
+24 |       operationId: exampleOne
+
+Error was generated by the assertions rule.
+
+
+[5] openapi.yaml:22:3 at #/paths/~1pet~1findByStatus
+
+Every path item must have a GET(required) operation.
+
+20 |   - name: Without description
+21 | paths:
+22 |   /pet/findByStatus:
+   |   ^^^^^^^^^^^^^^^^^
+23 |     put:
+24 |       operationId: exampleOne
+
+Error was generated by the assertions rule.
+
+
+[6] openapi.yaml:23:5 at #/paths/~1pet~1findByStatus/put
+
+Operation must not define both properties together: description and externalDocs
+
+21 | paths:
+22 |   /pet/findByStatus:
+23 |     put:
+   |     ^^^
+24 |       operationId: exampleOne
+25 |       summary: My resource
+
+Error was generated by the assertions rule.
+
+
+[7] openapi.yaml:23:5 at #/paths/~1pet~1findByStatus/put/x-codeSamples
+
+x-codeSamples must be defined
+
+21 | paths:
+22 |   /pet/findByStatus:
+23 |     put:
+   |     ^^^
+24 |       operationId: exampleOne
+25 |       summary: My resource
+
+Error was generated by the assertions rule.
+
+
+[8] openapi.yaml:25:7 at #/paths/~1pet~1findByStatus/put/summary
+
+Operation summary must have a maximum of 2 characters
+
+23 | put:
+24 |   operationId: exampleOne
+25 |   summary: My resource
+   |   ^^^^^^^
+26 |   description: summary example
+27 |   externalDocs:
+
+Error was generated by the assertions rule.
+
+
+[9] openapi.yaml:31:9 at #/paths/~1pet~1findByStatus/put/requestBody/content
+
+Only application/pdf can be used
+
+29 |   url: "https://example.com"
+30 | requestBody:
+31 |   content:
+   |   ^^^^^^^
+32 |     application/json:
```

Comment:
shouldn't the error location be on the wrong key instead of the on parent?

Reply:
It is not possible if we will have more than 1 wrong key. It's just current example has only 1.